src/fairchem/core/models/equiformer_v2/equiformer_v2.py

```diff
@@ -43,41 +44,43 @@
     SO3_Rotation,
 )
 from .transformer_block import (
-    FeedForwardNetwork,
-    SO2EquivariantGraphAttention,
     TransBlockV2,
 )
+from .weight_initialization import eqv2_init_weights
+
+with contextlib.suppress(ImportError):
+    pass
 
 if typing.TYPE_CHECKING:
     from torch_geometric.data.batch import Batch
 
-    from fairchem.core.models.base import GraphData
-
 # Statistics of IS2RE 100K
 _AVG_NUM_NODES = 77.81317
 _AVG_DEGREE = 23.395238876342773  # IS2RE: 100k, max_radius = 5, max_neighbors = 100
 
 
-def eqv2_init_weights(m, weight_init):
-    if isinstance(m, (torch.nn.Linear, SO3_LinearV2)):
-        if m.bias is not None:
-            torch.nn.init.constant_(m.bias, 0)
-        if weight_init == "normal":
-            std = 1 / math.sqrt(m.in_features)
-            torch.nn.init.normal_(m.weight, 0, std)
-    elif isinstance(m, torch.nn.LayerNorm):
-        torch.nn.init.constant_(m.bias, 0)
-        torch.nn.init.constant_(m.weight, 1.0)
-    elif isinstance(m, RadialFunction):
-        m.apply(eqv2_uniform_init_linear_weights)
+@deprecated(
+    "equiformer_v2_force_head (EquiformerV2ForceHead) class is deprecated in favor of equiformerV2_rank1_head  (EqV2Rank1Head)"
+)
+@registry.register_model("equiformer_v2_force_head")
+class EquiformerV2ForceHead(EqV2VectorHead):
+    def __init__(self, backbone):
+        logging.warning(
+            "equiformerV2_force_head (EquiformerV2ForceHead) class is deprecated in favor of equiformerV2_rank1_head  (EqV2Rank1Head)"
+        )
+        super().__init__(backbone)
 
 
-def eqv2_uniform_init_linear_weights(m):
-    if isinstance(m, torch.nn.Linear):
-        if m.bias is not None:
-            torch.nn.init.constant_(m.bias, 0)
-        std = 1 / math.sqrt(m.in_features)
-        torch.nn.init.uniform_(m.weight, -std, std)
+@deprecated(
+    "equiformer_v2_energy_head (EquiformerV2EnergyHead) class is deprecated in favor of equiformerV2_scalar_head  (EqV2ScalarHead)"
+)
+@registry.register_model("equiformer_v2_energy_head")
```

Comment:
should we just add `@registry.register_model("equiformer_v2_energy_head")`  on top of EqV2ScalarHead so that we dont need this extra class?

Reply:
oh ic, ok let's leave but lets remember to remove it in a future cleanup task then